hydromt_wflow/workflows/soilparams.py

```diff
@@ -5,12 +5,12 @@
 from hydromt import raster
 
 __all__ = [
-    "ksathorfrac",
+    "subsurface_ksat_horizontal_ratio",
```

Comment:
Maybe for the function name we can use only
```suggestion
    "ksat_horizontal_ratio",
```
And rename below ksat_vertical_vegetation?